app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -185,7 +185,7 @@ public function panel(Panel $panel): Panel
             ->userMenuItems([
                 MenuItem::make()
                     ->label('Profile Settings')
-                    ->url(fn () => EditProfile::getUrl())
+                    ->url(fn () => ProfileInformation::getUrl())
```

Comment:
@danharrin had mentioned that it was possible to just pass the URL from the Cluster class itself. Which would then go to the first page in the cluster.

Were you able to see if that was possible here?